From da5cd2688cf66bd2073f35d6f2a18b1e7d9c1775 Mon Sep 17 00:00:00 2001
From: Shammah Chancellor <shammah.chancellor@gmail.com>
Date: Wed, 28 Apr 2021 17:03:56 -0700
Subject: [PATCH 086/196] Remove existing seeds for Bitcoin ABC network (#160)

Since we are not part of BCHA, we need to remove the existing
seeds before launching mainnet.
---
 contrib/seeds/nodes_main.txt | 54 -------------------------------
 contrib/seeds/nodes_test.txt |  9 ------
 src/chainparamsseeds.h       | 63 ------------------------------------
 3 files changed, 126 deletions(-)

diff --git a/contrib/seeds/nodes_main.txt b/contrib/seeds/nodes_main.txt
index 5e7d6dda2..e69de29bb 100644
--- a/contrib/seeds/nodes_main.txt
+++ b/contrib/seeds/nodes_main.txt
@@ -1,54 +0,0 @@
-5.9.29.54:8333
-13.115.148.211:8333
-24.120.9.216:8333
-31.25.241.224:8339
-34.225.107.142:8333
-35.170.203.250:8333
-35.193.24.28:8333
-35.208.171.216:8333
-37.48.117.214:8333
-45.151.125.24:8333
-46.28.204.21:18505
-47.52.26.145:8333
-47.74.34.14:8333
-47.115.165.203:8333
-51.83.99.60:8333
-51.107.0.1:8333
-54.244.67.182:8333
-64.227.42.43:8333
-66.151.242.154:8339
-67.164.15.15:9333
-76.85.128.198:9333
-78.129.165.63:18328
-82.221.128.31:8364
-83.229.85.82:8333
-94.19.132.255:8339
-96.88.91.210:8333
-103.227.88.74:8333
-103.231.191.11:8533
-104.154.143.189:8333
-118.31.184.121:8333
-144.202.91.233:8333
-147.135.165.147:8338
-161.97.131.255:38333
-172.105.233.201:8333
-173.209.44.34:10333
-178.128.63.135:8333
-185.176.223.34:8333
-185.177.124.146:8333
-185.200.240.114:8332
-188.130.223.22:8333
-193.138.218.67:8333
-194.5.188.224:18798
-210.105.193.9:8333
-211.110.1.82:8333
-212.32.230.219:8333
-212.99.226.35:9060
-212.107.44.171:8333
-216.160.79.101:8333
-[2400:8902::f03c:91ff:fe17:3bf1]:8333
-[2a01:4f8:13a:4e::2]:9131
-[2a01:5f0:beef:5:0:3:0:1]:52102
-[2a02:1206:4547:56b0:e8c3:70bc:bd46:adc2]:8333
-[2a02:c207:2019:7777::1]:8333
-[2a03:1b20:1:f410:40::acef]:8333
diff --git a/contrib/seeds/nodes_test.txt b/contrib/seeds/nodes_test.txt
index 5cd71f82d..e69de29bb 100644
--- a/contrib/seeds/nodes_test.txt
+++ b/contrib/seeds/nodes_test.txt
@@ -1,9 +0,0 @@
-5.189.141.79:18333
-13.230.241.31:18333
-51.89.98.76:18333
-52.18.150.202:18333
-81.68.89.19:17333
-159.89.195.162:18333
-178.128.24.30:18333
-[2001:41d0:700:334c::]:18333
-[2a02:c207:2027:1503::1]:18333
diff --git a/src/chainparamsseeds.h b/src/chainparamsseeds.h
index f8489b0af..83d621683 100644
--- a/src/chainparamsseeds.h
+++ b/src/chainparamsseeds.h
@@ -8,71 +8,8 @@
  * IPv4 as well as onion addresses are wrapped inside an IPv6 address accordingly.
  */
 static SeedSpec6 pnSeed6_main[] = {
-    {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0x05,0x09,0x1d,0x36}, 8333},
-    {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0x0d,0x73,0x94,0xd3}, 8333},
-    {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0x18,0x78,0x09,0xd8}, 8333},
-    {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0x1f,0x19,0xf1,0xe0}, 8339},
-    {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0x22,0xe1,0x6b,0x8e}, 8333},
-    {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0x23,0xaa,0xcb,0xfa}, 8333},
-    {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0x23,0xc1,0x18,0x1c}, 8333},
-    {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0x23,0xd0,0xab,0xd8}, 8333},
-    {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0x25,0x30,0x75,0xd6}, 8333},
-    {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0x2d,0x97,0x7d,0x18}, 8333},
-    {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0x2e,0x1c,0xcc,0x15}, 18505},
-    {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0x2f,0x34,0x1a,0x91}, 8333},
-    {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0x2f,0x4a,0x22,0x0e}, 8333},
-    {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0x2f,0x73,0xa5,0xcb}, 8333},
-    {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0x33,0x53,0x63,0x3c}, 8333},
-    {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0x33,0x6b,0x00,0x01}, 8333},
-    {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0x36,0xf4,0x43,0xb6}, 8333},
-    {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0x40,0xe3,0x2a,0x2b}, 8333},
-    {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0x42,0x97,0xf2,0x9a}, 8339},
-    {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0x43,0xa4,0x0f,0x0f}, 9333},
-    {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0x4c,0x55,0x80,0xc6}, 9333},
-    {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0x4e,0x81,0xa5,0x3f}, 18328},
-    {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0x52,0xdd,0x80,0x1f}, 8364},
-    {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0x53,0xe5,0x55,0x52}, 8333},
-    {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0x5e,0x13,0x84,0xff}, 8339},
-    {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0x60,0x58,0x5b,0xd2}, 8333},
-    {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0x67,0xe3,0x58,0x4a}, 8333},
-    {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0x67,0xe7,0xbf,0x0b}, 8533},
-    {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0x68,0x9a,0x8f,0xbd}, 8333},
-    {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0x76,0x1f,0xb8,0x79}, 8333},
-    {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0x90,0xca,0x5b,0xe9}, 8333},
-    {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0x93,0x87,0xa5,0x93}, 8338},
-    {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0xa1,0x61,0x83,0xff}, 38333},
-    {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0xac,0x69,0xe9,0xc9}, 8333},
-    {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0xad,0xd1,0x2c,0x22}, 10333},
-    {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0xb2,0x80,0x3f,0x87}, 8333},
-    {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0xb9,0xb0,0xdf,0x22}, 8333},
-    {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0xb9,0xb1,0x7c,0x92}, 8333},
-    {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0xb9,0xc8,0xf0,0x72}, 8332},
-    {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0xbc,0x82,0xdf,0x16}, 8333},
-    {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0xc1,0x8a,0xda,0x43}, 8333},
-    {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0xc2,0x05,0xbc,0xe0}, 18798},
-    {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0xd2,0x69,0xc1,0x09}, 8333},
-    {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0xd3,0x6e,0x01,0x52}, 8333},
-    {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0xd4,0x20,0xe6,0xdb}, 8333},
-    {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0xd4,0x63,0xe2,0x23}, 9060},
-    {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0xd4,0x6b,0x2c,0xab}, 8333},
-    {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0xd8,0xa0,0x4f,0x65}, 8333},
-    {{0x24,0x00,0x89,0x02,0x00,0x00,0x00,0x00,0xf0,0x3c,0x91,0xff,0xfe,0x17,0x3b,0xf1}, 8333},
-    {{0x2a,0x01,0x04,0xf8,0x01,0x3a,0x00,0x4e,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x02}, 9131},
-    {{0x2a,0x01,0x05,0xf0,0xbe,0xef,0x00,0x05,0x00,0x00,0x00,0x03,0x00,0x00,0x00,0x01}, 52102},
-    {{0x2a,0x02,0x12,0x06,0x45,0x47,0x56,0xb0,0xe8,0xc3,0x70,0xbc,0xbd,0x46,0xad,0xc2}, 8333},
-    {{0x2a,0x02,0xc2,0x07,0x20,0x19,0x77,0x77,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x01}, 8333},
-    {{0x2a,0x03,0x1b,0x20,0x00,0x01,0xf4,0x10,0x00,0x40,0x00,0x00,0x00,0x00,0xac,0xef}, 8333}
 };
 
 static SeedSpec6 pnSeed6_test[] = {
-    {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0x05,0xbd,0x8d,0x4f}, 18333},
-    {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0x0d,0xe6,0xf1,0x1f}, 18333},
-    {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0x33,0x59,0x62,0x4c}, 18333},
-    {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0x34,0x12,0x96,0xca}, 18333},
-    {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0x51,0x44,0x59,0x13}, 17333},
-    {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0x9f,0x59,0xc3,0xa2}, 18333},
-    {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0xb2,0x80,0x18,0x1e}, 18333},
-    {{0x20,0x01,0x41,0xd0,0x07,0x00,0x33,0x4c,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00}, 18333},
-    {{0x2a,0x02,0xc2,0x07,0x20,0x27,0x15,0x03,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x01}, 18333}
 };
 #endif // BITCOIN_CHAINPARAMSSEEDS_H
-- 
2.39.5 (Apple Git-154)

